Try to add clarity about what a transmittal notice is

flooose opened the pull request:
We've had another [AWS Access Request with PII in it](https://github.com/department-of-veterans-affairs/va.gov-team/issues/72319#issuecomment-1869678794). This PR is intended to help prevent that.

For a preview of the change, see [file at the last commit - 45d55c0cb26bf189167717b9fb58165c2e03b285](https://github.com/department-of-veterans-affairs/va.gov-team/blob/45d55c0cb26bf189167717b9fb58165c2e03b285/.github/ISSUE_TEMPLATE/aws-access-request.yml).

Adds a note about:
- That PDF documents are usually their personal documents **with PII** in them.
- The "subject" of a transmittal email

Removed:
- Requirement for an E-QIP transmittal date, which should now be covered by the checks and warnings about the "E-QIP Transmittal Confirmation"

flooose committed:
Try to add clarity about what a transmittal notice is

flooose committed:
Remove redundancy in description

The line that was removed reflects something that gets mentioned in
the `placeholder` field.

flooose committed:
Make note about PII more noticeable

rmtolmach commented on .github/ISSUE_TEMPLATE/aws-access-request.yml:
```diff
@@ -109,7 +109,10 @@ body:
+        **NOTE: Do not submit PDF copies of your E-QIP document! These usually contain your social security number and other PII**
+
+        Your transmittal confirmation should contain the words "Transmittal Notice" in the subject of the email. If it doesn't, your E-QIP has not begun processing on the VA side.
       placeholder: Drag your transmittal confirmation email screenshot here
```
Could we say something here that reiterates what the screenshot should be? 
<img width="297" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/10993987/08bbc7ce-a3fb-4fb3-aca2-747a8e423096">

"Drag or paste your transmittal confirmation EMAIL screenshot here. Your transmittal confirmation should contain the words "Transmittal Notice" in the subject of the email. If it doesn't, your E-QIP has not begun processing on the VA side."

idk, that seems like a lot 😆

rmtolmach commented on .github/ISSUE_TEMPLATE/aws-access-request.yml:
```diff
@@ -109,7 +109,10 @@ body:
       label: E-QIP Transmittal Confirmation
-      description: Attach a screenshot of your transmittal confirmation email by clicking this area to highlight and dragging the file in.
+      description: |
+        **NOTE: Do not submit PDF copies of your E-QIP document! These usually contain your social security number and other PII**
```
Are emojis allowed in this section of the template? If so, maybe a ⚠️?

flooose committed:
Update formatting and placements of notes

flooose committed:
Remove requirement for eqip transmittal date

This should be covered by the checks and warnings in the "E-QIP
Transmittal Confirmation"

rmtolmach reviewed (approved):
Looks good! Thanks for making this more clear!
